client/landing/stepper/declarative-flow/onboarding.ts

```diff
@@ -117,8 +117,7 @@ const onboarding: Flow = {
 							return;
 
 						case SiteIntent.DIFM:
-							// TODO Implement exit to DIFM
-							return;
+							return window.location.assign( '/start/do-it-for-me' );
```

Comment:
We don't have to do it in this PR, but something we need to keep aware of: users might be logged out which means the locale is passed in the URL. Not something we usually have to think about, but we do now that so much is happening before the signup step.

The Spanish onboarding flow starts at `/setup/onboarding/es`. And if that's the case then the link needs to go to `/start/do-it-for-me/es`